pip/commands/show.py

```diff
@@ -109,9 +109,9 @@ def search_packages_info(query):
         classifiers = []
         for line in metadata.splitlines():
             if not line:
-                break
+                pass  # The Problem SJB
```

Comment:
The `if not line` makes sure that line is not `None`, blank lines are returned from splitlines as type None and this will raise an `AttributeError` on the `line.startswith()` - I preferred leaving the `if not line` and then using `elseif` to using a compound `if line and line.startswith()` for clarity and in case future enhancements need to detect other cases.